Dockerfile

```diff
@@ -1,8 +1,9 @@
-FROM maven:3.6.3-jdk-11
+FROM maven:3.8.5-openjdk-17-slim
 
 ADD entrypoint.sh /entrypoint.sh
 ADD add-comment.rb /add-comment.rb
 ADD Gemfile /Gemfile
+ADD maven-settings.xml /maven-settings.xml
```

Comment:
❓ Could you detail what we need the settings for? Is there a specific failure we're fixing with this? I am wondering if we need to adjust all consumers of this action as a follow-up (not sure how many repositories that actually are).

Reply:
Didn't find a strong argument for it so I will revert the changes for maven-settings.